static/js/reducers/index.js

```diff
@@ -23,6 +27,48 @@ import {
   FETCH_SUCCESS,
 } from '../actions';
 
+import { HIGH_SCHOOL, ASSOCIATE, BACHELORS, MASTERS, DOCTORATE } from '../constants';
+
+export const INITIAL_DIALOG_STATE = {
+  openDialog: false,
+  degreeLevel: '',
+  educationIndex: null,
+  educationId: null
+};
+export const educationDialog = (state = INITIAL_DIALOG_STATE, action) => {
+  switch (action.type) {
+  case SHOW_EDUCATION_FORM_DIALOG:
+    return Object.assign({}, state, {
+      openDialog: true,
+      degreeLevel: action.payload.level,
+      educationIndex: action.payload.index
+    });
+  case HIDE_EDUCATION_FORM_DIALOG:
+    return INITIAL_DIALOG_STATE;
+  default:
+    return state;
+  }
+};
+let education_level_state = {};
+education_level_state[HIGH_SCHOOL] = true;
+education_level_state[ASSOCIATE] = true;
+education_level_state[BACHELORS] = true;
+education_level_state[MASTERS] = false;
+education_level_state[DOCTORATE] = false;
+
+export const INITIAL_EDUCATION_LEVEL_STATE = education_level_state;
+
+export const educationLevels = (state = INITIAL_EDUCATION_LEVEL_STATE, action) => {
+  switch (action.type) {
+  case TOGGLE_EDUCATION_LEVEL:
+    return Object.assign({}, state, action.payload.newState);
```

Comment:
Since `newState` contains the entire set of education levels, we could just `return action.payload.newState` here instead